style.css

```diff
@@ -60,61 +64,51 @@ main {
 /* Headline Section Styling */
 
 #headline-container {
-  background-image: url("./images/header-illsutration-mobile@2x.png");
+  background-image: url("./images/header-illsutration-mobile.svg");
   background-repeat: no-repeat;
-  background-size: 100%;
-  width: 95vw;
+  background-size: cover;
+  width: 95%;
```

Comment:
- [ ] Your headline section is not perfectly centered on the desktop page. You could use 100% as the width, with 95%, the space on the right is larger than the space on the left of this section making it not centered on the page. Kindly fix it.